installer/PowerToysSetup/Product.wxs

```diff
@@ -14,6 +14,14 @@
     <Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine" InstallPrivileges="elevated" Platform="x64" />
 
     <MajorUpgrade DowngradeErrorMessage="A later version of [ProductName] is already installed." />
```

Comment:
Just a thought since we're discussing MSI upgrades:

```suggestion
     <MajorUpgrade DowngradeErrorMessage="A later version of [ProductName] is already installed." Schedule="afterInstallInitialize" />
```

This change will instruct Windows to restore the previous version if installing the new version fails for some reason. [Here are the docs for this element](https://wixtoolset.org/documentation/manual/v3/xsd/wix/majorupgrade.html), if you want to see them. I don't know why this isn't put in the default template, honestly.

Reply:
OK, feel free to open a PR for this change. Thanks!